_pages/assignments.html

```diff
@@ -23,6 +23,14 @@
     <div class="col-md-1 col-1 col-xs-0"></div>
   	<div class="col-md-10 col-10 col-xs-12">
       <h1>Assignments</h1>
+      <section class="infrastructure-links">
+        <p>
+          <a href="https://sp19-cs241-grd-02.cs.illinois.edu/on-demand/" class="btn btn-success" role="button">Broadway On-Demand Autograder</a>
```

Comment:
I really don't like these hardcoded links here... again not something that needs to be fixed in this PR, but maybe they could be replaced by templates that can inject the current semester's url?

Reply:
Done.